app/services/publish.js

```diff
@@ -51,10 +62,12 @@ export default class PublishService extends Service {
       this.store.query('behandeling-van-agendapunt', {
         'filter[onderwerp][zitting][:id:]': meetingId,
         include: 'onderwerp',
+        page: { size: 1000 },
```

Comment:
this will do for now, but requesting page per page would be my recommended solution to avoid request timeouts

Reply:
yeah I know
we really need to abstract that away at some point
it's way too much of a pain to add every time